chromium_src/base/threading/thread_restrictions.h

```diff
@@ -0,0 +1,16 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_CHROMIUM_SRC_BASE_THREADING_THREAD_RESTRICTIONS_H_
+#define BRAVE_CHROMIUM_SRC_BASE_THREADING_THREAD_RESTRICTIONS_H_
+
+class BraveBrowsingDataRemoverDelegate;
+
+#define BRAVE_SCOPED_ALLOW_BASE_SYNC_PRIMITIVES_H \
+  friend class ::BraveBrowsingDataRemoverDelegate;
+#include "../../../../base/threading/thread_restrictions.h"
```

Comment:
Nit: empty line after a multi-line `#define`

Reply:
force pushed to add an empty line after define for this file and `chromium_src/chrome/browser/browsing_data/chrome_browsing_data_remover_delegate.h`